index.bs

```diff
@@ -376,42 +408,75 @@ To <dfn>parse an attribution destination</dfn> from a string |str|:
 
 <h3 algorithm id="obtaining-attribution-source-anchor">Obtaining an attribution source from an <code>a</code> element</h3>
 
-To <dfn>obtain an attribution source</dfn> from an <{a}> element |anchor|:
+To <dfn>obtain an attribution source from an anchor</dfn> given an <{a}> element |anchor|:
+
+1. Let |params| be an {{AttributionSourceParams}} with the following key/value pairs:
```

Comment:
This could follow example 33 https://heycam.github.io/webidl/#idl-dictionaries if we wanted and just send an ordered map in rather than directly doing this construction. I'm having a hard time finding examples of constructing IDL dictionaries as input to a shared function

@jyasskin is there an accepted practice here?